addon/adapters/osf-adapter.js

```diff
@@ -85,8 +85,10 @@ export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
                 return this._super(...arguments).then(response => Ember.RSVP.allSettled(promises).then(() => response));
             }
             return this._super(...arguments);
-        } else {
+        } else if (promises) {
             return Ember.RSVP.allSettled(promises).then(() => null);
+        } else {
+            return new Ember.RSVP.Promise((resolve) => resolve(null));
```

Comment:
It seems kind of strange to me that we have to check for this- shouldn't `updateRecord` already know the model is clean and do nothing? (is this a sign that we're skipping some basic machinery of the method?)